src/pages/run-a-node.js

```diff
@@ -0,0 +1,1011 @@
+              <h3>
+                <StyledEmoji text=":shopping_cart:" size={2} />
+                <Translation id="page-run-a-node-buy-fully-loaded-title" />
+              </h3>
+              <p>
+                <Translation id="page-run-a-node-buy-fully-loaded-description" />
+              </p>
+              <ul>
+                <li>
+                  <Translation id="page-run-a-node-buy-fully-loaded-note-1" />
+                </li>
+                <li>
+                  <Translation id="page-run-a-node-buy-fully-loaded-note-2" />
+                </li>
+                <li>
+                  <code>
+                    <Translation id="page-run-a-node-buy-fully-loaded-note-3" />
+                  </code>
+                </li>
+              </ul>
+            </div>
+            <ScrollButtonSecondary
+              onClick={() => scrollToId("choose-your-adventure")}
+            >
+              <Translation id="page-run-a-node-shop" />
+            </ScrollButtonSecondary>
+          </FullyLoaded>
+        </MarginFlex>
+
+        <h3 id="plan-on-staking">
+          <StyledEmoji text=":cut_of_meat:" size={2} />
+          <Translation id="page-run-a-node-staking-plans-title" />
+        </h3>
+        <p>
+          <Translation id="page-run-a-node-staking-plans-description" />
+        </p>
+        <p>
+          <Link to="https://cpubenchmark.net">cpubenchmark.net</Link>
+        </p>
+        <h3 id="rasp-pi">
+          <StyledEmoji text=":pie:" size={2} />
+          <Translation id="page-run-a-node-rasp-pi-title" />
+        </h3>
+        <p>
+          <Translation id="page-run-a-node-rasp-pi-description" />
+        </p>
+        <ul>
+          <li>
+            <Link to="https://docs.dappnode.io/get-started/installation/arm-hardware/installation">
+              <Translation id="page-run-a-node-rasp-pi-note-1-link" />
+            </Link>{" "}
+            -{" "}
+            <i>
+              <Translation id="page-run-a-node-rasp-pi-note-1-description" />
+            </i>
+          </li>
+          <li>
+            <Link to="https://ethereum-on-arm-documentation.readthedocs.io/en/latest">
+              <Translation id="page-run-a-node-rasp-pi-note-2-link" />
+            </Link>{" "}
+            -{" "}
+            <i>
+              <Translation id="page-run-a-node-rasp-pi-note-2-description" />
+            </i>
+          </li>
+          <li>
+            <Link to="/developers/tutorials/run-node-raspberry-pi">
+              <Translation id="page-run-a-node-rasp-pi-note-3-link" />
+            </Link>{" "}
+            -{" "}
+            <i>
+              <Translation id="page-run-a-node-rasp-pi-note-3-description" />
+            </i>
+          </li>
+        </ul>
+      </Content>
+
+      <Content>
+        <h2>
+          <Translation id="page-run-a-node-getting-started-software-title" />
+        </h2>
+        <GappedContent>
+          <SoftwareHighlight backgroundColor="homeBoxTurquoise">
```

Comment:
This section feels a bit cramped to me. One thought I had was maybe making it a bit narrower similar to the '[How to stake](https://ethereum.org/en/eth2/staking)' section on our staking page?

**Proposed**
![Screenshot 2022-01-07 at 23 42 07](https://user-images.githubusercontent.com/62268199/148620839-0342e241-d90b-4646-962d-6da136ffac1e.png)

**Current**
![Screenshot 2022-01-07 at 23 42 15](https://user-images.githubusercontent.com/62268199/148620834-99ca53fe-d805-498e-b957-ed832b06ef47.png)

Reply:
Don't love this solution but 🤷‍♂️

Reply:
Yeah wouldn't be against adding some horizontal margin on those divs when on suitable screen sizes.